commercial/app/model/merchandise/Merchandise.scala

```diff
@@ -203,35 +210,27 @@ object Masterclass {
       }
     }
 
-    def extractFirstParagraph(html: String) = {
-      val firstParagraph: Option[Element] = Some(doc.select("p").first())
-      firstParagraph match {
-        case Some(p) => p.text
-        case _ => ""
-      }
-    }
-
     extractGuardianUrl map { extractedUrl =>
 
       new Masterclass(
         id = event.id,
         name = event.name,
-        startDate = event.startDate,
+        startDate = event.start_date,
```

Comment:
So, as you alluded to above, I altered the `Eventbrite` model to reflect the Eventbrite feed values  to allow us to utilise the Json macros. In the case of `Eventbrite.Event`, we have an explicit `Reads` method so it doesn't actually help here, but in other cases, such as `Eventbrite.Pagination` and `Eventbrite.Response` the snake-case helps us out by letting Play read it implicitly to our case class. I decided therefor to match all the Eventbrite case class variables to their real-feed names, type and all. 

If you reckon that this isn't worthwhile then I'm happy to take it out and write explicit reads, just thought I'd throw it in there :)